src/main/java/ch/njol/skript/expressions/ExprPlayerlistHeaderFooter.java

```diff
@@ -44,17 +44,17 @@
 @Description("The message above and below the player list in the tab menu.")
 @Examples({"set all players tab list header to \"Welcome to the Server!\"",
 			"send \"%the player's tab list header%\" to player",
-			"reset all player's tab list header"})
+			"reset all players' tab list header"})
 @Since("2.4")
-@RequiredPlugins("Spigot 1.13 or newer")
+@RequiredPlugins("Minecraft 1.13 or newer")
 public class ExprPlayerlistHeaderFooter extends PropertyExpression<Player, String> {
 	
 	static {
 		if (Skript.methodExists(Player.class, "setPlayerListHeaderFooter", String.class, String.class)) //This method is only present if the header and footer methods we use are
-			Skript.registerExpression(ExprPlayerlistHeaderFooter.class, String.class, ExpressionType.PROPERTY, "[the] %players% (player|tab) list (1¦header|2¦footer)");
+			Skript.registerExpression(ExprPlayerlistHeaderFooter.class, String.class, ExpressionType.PROPERTY, "[the] %players%['][s] (player|tab)[ ]list (header|1¦footer)[s] [(text|message)[s]]");
```

Comment:
```suggestion
			PropertyExpression.register(ExprPlayerlistHeaderFooter.class, String.class, "(player|tab)[ ]list (header|1¦footer) [(text|message)]", "players");
```

Reply:
Why did you remove the plurality of the words? `player list header of all players`